gnovm/pkg/gnolang/machine.go

```diff
@@ -677,7 +679,7 @@ func (m *Machine) RunFunc(fn Name) {
 	defer func() {
 		if r := recover(); r != nil {
 			fmt.Printf("Machine.RunFunc(%q) panic: %v\n%s\n",
-				fn, r, m.String())
+				fn, r, m.Stacktrace())
```

Comment:
Why does this call `Stacktrace` instead of `ExceptionsStacktrace`?

Reply:
my bad something I forget to revert. Corrected here: https://github.com/gnolang/gno/pull/2145/commits/7cf4c7658de44b348989a8a175a18d4b4577fc2d